resources/lib/videoplayer.py

```diff
@@ -281,20 +236,14 @@ def _ask_to_skip(self, section):
         # show only for the first X seconds
         dialog_duration = min(dialog_duration, self._skip_modal_duration_max)
 
-        threading.Thread(
-            target=_show_modal_dialog,
-            args=[
-                SkipModalDialog,
-                "plugin-video-crunchyroll-skip.xml"
-            ],
-            kwargs={
+
+        show_modal_dialog(SkipModalDialog, "plugin-video-crunchyroll-skip.xml",**{
```

Comment:
Not sure about this, this is supposed to run in the background in a separate thread to not block things. At least this is how it's done in the Netflix addon.

Reply:
I have not noticed any problems here.
I can also undo this if you wish.